recipes/upstream/aws/k3s/README.md

```diff
@@ -11,77 +11,30 @@ git clone https://github.com/rancherlabs/tf-rancher-up.git
-    -  uncomment `instance_type` and change the instance type if needed.
     -  `prefix` to give the resources an identifiable name (eg, your initials or first name)
-    -  Recommended: `spot_instances` can be set to `true` to use spot instances
-- Check your AWS credentials are configured in `~/.aws/credentials`, terraform will use these by default. Refer the [`aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-methods) command on how to do this.
-- If you don't want to configure AWS credentials using `aws configure` in above step, uncomment `aws_access_key` and `aws_secret_key` in `terraform.tfvars` and input the required keys there.
-- If an HA cluster need to be deployed, change the `instance_count` variable to 3 or more.
-- There are more optional variables which can be tweaked under `terraform.tfvars`.
-
-**NOTE** you may need to use ` terraform init -upgrade` to upgrade provider versions
-
-Execute the below commands to start deployment.
-
-```bash
-terraform init
-terraform plan -var-file terraform.tfvars
-terraform apply -var-file terraform.tfvars
-```
-The login details will be displayed in the screen once the deployment is successful. It will have the details as below.
-
-```bash
-rancher_hostname = "https://rancher.<xx.xx.xx.xx>.sslip.io"
-rancher_password = "initial-admin-password"
-```
+    -  `aws_region` to suit your region
+    -  `server_nodes_count` to specify the number of Master nodes to create (to maintain ETCD quorum, the value must be 1, 3, or 5)
+    -  `worker_nodes_count` to specify the number of Worker nodes to create
+    -  `ssh_username` to specify the user used to create the VMs (default "ubuntu")
+    -  `rancher_hostname` in order to reach the Rancher console via DNS name
+    -  `rancher_password` to configure the initial Admin password (the password must be at least 12 characters)
+- Make sure you are logged into your AWS Account from your local Terminal. See the preparatory steps [here](../../../../modules/infra/aws/README.md).
 
-- If storing multiple AWS credentials in `~/.aws/credentials`, set the profile when running terraform.
+**NB: If you want to use all the configurable variables in the `terraform.tfvars` file, you will need to uncomment them there and in the `variables.tf` and `main.tf` files.**
 
 ```bash
-AWS_PROFILE=<profile name> terraform plan -var-file terraform.tfvars
-AWS_PROFILE=<profile name> terraform apply -var-file terraform.tfvars
+terraform init -upgrade ; terraform apply -target=module.k3s-first-server.tls_private_key.ssh_private_key -target=module.k3s-first-server.local_file.private_key_pem -target=module.k3s-first-server.local_file.public_key_pem -target=module.k3s-first-server.aws_key_pair.key_pair -target=module.k3s-first-server.aws_vpc.vpc -target=module.k3s-first-server.aws_subnet.subnet -target=module.k3s-first-server.aws_security_group.sg_allowall -auto-approve ; terraform apply -auto-approve
```

Comment:
Let's explore if there is a way to avoid such a long `terraform apply`.

Reply:
@leodotcloud, fixed as requested; now everything works with a single command.

Reply:
cc @ajithijk , @dnoland1